docs/pages/inboxes/pick-an-sdk.md

```diff
@@ -0,0 +1,21 @@
+# Pick an SDK
+
+Choose the appropriate XMTP SDK based on your app platform and development needs.
+
+## Web
+
+These SDKs are designed for web-based app, offering seamless integration with browser and server environments.
+
+- [Browser SDK](https://github.com/xmtp/xmtp-js/tree/main/sdks/browser-sdk)
+
+- [Node SDK](https://github.com/xmtp/xmtp-js/tree/main/sdks/node-sdk)
```

Comment:
i'm not sure how to classify this SDK since it can be completely separate from the Web. perhaps `Server` is the right category?